packages/shared/src/UI/components/SettingsBoard/GasForm.tsx

```diff
@@ -78,34 +100,43 @@ export function GasForm(props: GasFormProps) {
         'maxFeePerGas',
         'maxPriorityFeePerGas',
     ])
+    const [gasPriceByUser, setGasPriceByUser] = useState<string>()
+    const [maxFeePerGasByUser, setMaxFeePerGasByUser] = useState<string>()
+    const [maxPriorityFeePerGasByUser, setMaxPriorityFeePerGasByUser] = useState<string>()
 
-    const error = useMemo(() => {
+    const errorCenter = useMemo(() => {
```

Comment:
Access `methods.formState.errors` many times.
```suggestion
    const errors = methods.formState.errors
    const errorCenter = useMemo(() => {
```